scripts/setup.sh

```diff
@@ -23,7 +23,11 @@ export OS=`uname`
 export MACHINE_TYPE=`uname -m`
 export TOOLS_DIR=./tools
 export NODE_DIR=$TOOLS_DIR/node-6.9.1
-export NPM_CMD=$NODE_DIR/bin/npm
+if [ ${OS}==MINGW_NT-10.0 ]; then
```

Comment:
Shouldn't, since the PATH on line 33 references $NODE_DIR instead of $NODE_CMD. $NODE_DIR is the same regardless of OS, but $NPM_CMD is different in the windows distro of node, which doesn't like to use /bin directories for some reason.